sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -135,4 +135,90 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
     const unwrappedText = uint8ArrayToString(unwrappedResult.result);
     assert.equal(text, unwrappedText);
   });
+
+  if (isRecordMode() || process.env.TEST_MODE === "live") {
```

Comment:
This block looks fine, but you might consider making whether to use HSM a boolean variable and run it with both `true` and `false` to always run the exact same RSA and RSA-HSM tests. Don't forget about EC and EC-HSM as well.

Reply:
We don't have local crypto for EC operations, so we don't have code for EC and EC-HSM. Should I add tests anyway?